.github/workflows/docker.yml

```diff
@@ -17,7 +17,15 @@ jobs:
           sudo rm -rf /usr/share/dotnet /usr/local/lib/android /opt/ghc
           echo "after freeing up space:"
           df -h
-     # Workaround for https://github.com/rust-lang/cargo/issues/8719
+      - name: Determine Docker image metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: eqlabs/pathfinder
+          tags: |
+            type=semver,pattern={{raw}}
```

Comment:
Does raw mean `git describe --tags` output or something else?

Reply:
I don't quite follow you here. What's "latest tag on main" here?

We only trigger this workflow automatically for `v*` Git tags being pushed to the repository. For that event the line above will produce a Docker tag value that matches the name of the Git tag.

Reply:
> What's "latest tag on main" here?

Well at the moment I'd say it's `v0.2.1-alpha`. So if this always is the latest tag reachable from the commit, I would expect it to re-push the built image as the latest tag (which we wouldn't want).

Looking through the docs I understand that for `type=semver` it requires a push action:
> Will be used on a push tag event and requires a valid semver Git tag

I missed this the first time around. The next line's `type=raw` is used unconditionally. This configuration seems to make sense. Okay to resolve.